resources/types/Tree.yaml

```diff
@@ -0,0 +1,16 @@
+global-object-type: Tree
+storage-type: WS
+storage-object-type: KBaseTrees.Tree
+versions:
+- indexing-rules:
+  - path: default_node_labels/{size}
+    keyword-type: integer
+    key-name: nodes
+    ui-name: Nodes
+  - path: type
+    keyword-type: string
+    ui-name: Tree Type
+  - path: default_node_labels/*/
```

Comment:
How many labels might there be? If there are thousands this might cause problems

Reply:
I thought that there might be a maximum size, but the ES docs don't mention one: https://www.elastic.co/guide/en/elasticsearch/reference/5.5/array.html

So maybe it's ok. There will be a java limit of ~2B...

Of course there are indexing performance and search performance implications.